cmake/modules/patch-dpdk-conf.sh

```diff
@@ -17,6 +17,8 @@ machine=$1
 shift
 arch=$1
 shift
+numa=$1
```

Comment:
we will need a better way to pass the parameters if the list gets longer over time.